bridge/checkpoint/transport/proto/transport.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package v1;
+
+option go_package = "/bridge/checkpoint/transport/proto";
+
+message SignedMessage {
```

Comment:
@dbrajovic I noticed you can use same name among all protoc in project. Can you rename this to some other name?

Reply:
What's wrong with having this one ? It's encapsulated within the `checkpoint` module and represents the same concept used in `statesync`. 

Other than that, I'm open to naming suggestions.

Reply:
@dbrajovic For now, you can see this error on building. Please check it